src/index.js

```diff
@@ -0,0 +1,14 @@
+const createThemeProvider = require('./create-theme-provider');
+const createWithTheme = require('./create-with-theme');
+const channel = require('./channel');
+
+module.exports = {
+  channel,
+  withTheme: createWithTheme(),
+  ThemeProvider: createThemeProvider(),
+  createTheming: (customChannel = channel) => ({
```

Comment:
its a question if you are using the context or not. theming is the entire package, createTheming is  basically theming.createTheming instead of theming.create